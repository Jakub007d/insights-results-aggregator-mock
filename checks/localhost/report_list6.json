{
	"clusters": null,
	"errors": null,
	"reports": {}
}
