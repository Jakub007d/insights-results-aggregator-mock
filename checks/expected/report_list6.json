{
	"clusters": null,
	"errors": null,
	"reports": {},
	"generated_at": "2020-08-11T10:17:29Z"
}